flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/plan/batch/sql/TableScanTest.scala

```diff
@@ -171,4 +171,38 @@ class TableScanTest extends TableTestBase {
         .createTemporaryTable("t1")
     util.verifyPlan(util.tableEnv.from("t1"))
   }
+
+  @Test
+  def testCreateViewWithRowNumber(): Unit = {
+    util.addTable(
+      """
+        |CREATE TABLE test_source (
+        |  name STRING,
+        |  eat STRING,
+        |  age BIGINT
+        |) WITH (
+        |  'connector' = 'values',
+        |  'bounded' = 'true'
```

Comment:
Could you use `bounded = false` ? This can verify whether the Rank operator is generated.